### ISSUE TITLE
7.5 release schedule (subscribe to help or for updates)

### ISSUE THREAD

**Issue by Carreau** [ipython/ipython#11657]
This is the release check list for 7.5 release. It is quite a bit of work.
There are some task that need to be done by core-dev, but many task that could be done by contributor to help. Task are not necessarily ordered and some can be done before others.

Subscribe if you want to help I'll update the issue as I'm working toward release. I usually attempt to do monthly release between 20 and 28 when I have time. The more help I get, the more probable the releases are :-)

 - [x] Write what's new and round up issues. Make sure all closed issues/PR are correctly tagged .
 - [x] make the release and pushed 7.5 on GitHub (Needs Core-dev), remember to sign it. i.e follow instructions [there](https://ipython.readthedocs.io/en/stable/coredev/index.html#ipython-release-process).
 - [x] uploaded to PyPI (Needs Core-dev)
 - [x] update read the docs stable tag :  automatic, stable is always the latest tag.
 - [x] **Help Welcomed**: Update the ipython website (https://github.com/ipython/ipython-website), see for example https://github.com/ipython/ipython-website/commit/9c021b5da786031506aa89110605fb94d105b251 and do the same to annouce the release and release date of 7.5
 - [x] **Help Welcomed**: Update the wikipedia page to point to latest release and its date.
 - [x] **Help Welcomed, should be automatic**: Update conda forge feedstock (https://github.com/conda-forge/ipython-feedstock) with new version number and SHA-hash. 
 - [ ] **Help Welcomed**: Make an announcement on the mailing list.
 - [x] **Help Welcomed**: Make an announce on Discourse (example : https://discourse.jupyter.org/t/release-of-ipython-7-2/226, https://discourse.jupyter.org/t/release-of-ipython-7-3/409)
 - [x] **Help Welcomed**: Make an announce on Twitter (need SO with access to twitter account)  (example https://twitter.com/Mbussonn/status/1097617783620198400)
 

I'm happy to delegate anything marked *Help Welcomed* to someone who want to help contribute to IPython and want to engage. I'm happy to details the subtasks more for clarification.

Previous release tasks: https://github.com/ipython/ipython/issues/11614

```
$ shasum -a 256 dist/*
54c5a8aa1eadd269ac210b96923688ccf01ebb2d0f21c18c3c717909583579a8  dist/ipython-7.5.0-py3-none-any.whl
e840810029224b56cd0d9e7719dc3b39cf84d577f8ac686547c8ba7a06eeab26  dist/ipython-7.5.0.tar.gz
```

**Comment by Carreau** [comment]
@meeseeksdev tag "help wanted"

**Comment by Carreau** [comment]
@meeseeksdev party

**Comment by Carreau** [comment]
Released a small 7.5 on PyPI. Will take care of the rest later unless anyone else is interested in helping/doing it.

**Comment by gpotter2** [comment]
See https://github.com/ipython/ipython-website/pull/147
